G_channels.md

```diff
@@ -1,4 +1,4 @@
-Channels are a really exciting and powerful part of Phoenix, one that allows us to easily add soft-realtime features to our applications. Channels are based on a simple idea - sending and receiving messages. Senders broadcast messages about topics. Receivers subscribe to topics so that they can get those messages. Senders and receivers can switch roles on the same topic at any time.
+Channels are a really exciting and powerful part of Phoenix that allows us to easily add soft-realtime features to our applications. Channels are based on a simple idea - sending and receiving messages. Senders broadcast messages about topics. Receivers subscribe to topics so that they can get those messages. Senders and receivers can switch roles on the same topic at any time.
```

Comment:
Interesting. I read the sentence as, "Channels allow us . . . " with "Channels" as the subject and "allow" as the verb. I also read the "are a powerful part of Phoenix that" as an adjectival clause describing "Channels".

Does that make sense, or should I write an angry letter to my 8th grade English teacher? ;^) ;^) ;^)